zerver/migrations/0497_archivedmessage_has_reaction_message_has_reaction.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.9 on 2024-02-27 15:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0496_alter_scheduledmessage_read_by_sender"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="archivedmessage",
+            name="has_reaction",
+            field=models.BooleanField(db_index=True, default=False),
+        ),
+        migrations.AddField(
+            model_name="message",
+            name="has_reaction",
+            field=models.BooleanField(db_index=True, default=False),
+        ),
+    ]
```

Comment:
You have one more major bug in this code, which is that you need to write a backfill migration that 

Read `zerver/migrations/` to understand some examples; `ls zerver/migrations/*backfill*` should give several useful examples.